solr/licenses/bcutil-LICENSE-MIT.txt

```diff
@@ -0,0 +1,15 @@
+Copyright (c) 2000-2010 The Legion Of The Bouncy Castle (http://www.bouncycastle.org)
```

Comment:
https://www.bouncycastle.org/license.html has updated dates (up to 2021). Not sure how much this matters.